lib/twiliosms.rb

```diff
@@ -0,0 +1,32 @@
+require 'twilio-ruby'
+require 'date'
+
+class TwilioSMS
+  DELIVERY_TIME = 1
+
+  attr_reader :delivery_time
+
+  def initialize(account_sid = ENV["TWILIO_ACCOUNT_SID"], auth_token = ENV["TWILIO_AUTH_TOKEN"])
+    @account_sid = account_sid
+    @auth_token = auth_token
+
+    @delivery_time = Time.now.strftime("%H:%M").split(":")
+    @delivery_time[0] = @delivery_time[0].to_i
+    @delivery_time[0] += DELIVERY_TIME
+    @delivery_time[0] = @delivery_time[0].to_s
+    @delivery_time = @delivery_time.join(":")
```

Comment:
I'm not sure it makes sense that the TwillioSMS class is the one calculating the delivery time. Maybe the class that calls `send_sms` could pass in a delivery time instead?

Reply:
Or maybe it's just the class that is not well name, and this should be called `DeliveryNotifier` or something.